store/types/types.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package cosmos_sdk.store.v1;
```

Comment:
nit, this should live in the `proto/` directory

Reply:
Good call, thanks - this has changed a bit since the PR was first submitted back in March. :)